litt/littdb_config.go

```diff
@@ -0,0 +1,196 @@
+package litt
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"math/rand"
+	"time"
+
+	"github.com/Layr-Labs/eigenda/common"
+	"github.com/Layr-Labs/eigenda/litt/disktable/keymap"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/docker/go-units"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Config is configuration for a litt.DB.
+type Config struct {
+	// The context for the database. If nil, context.Background() is used.
+	CTX context.Context
+
+	// The paths where the database will store its files. If the path does not exist, it will be created.
+	// If more than one path is provided, then the database will do its best to spread out the data across
+	// the paths. If the database is restarted, it will attempt to load data from all paths. Note: the number
+	// of paths should not exceed the sharding factor, or else data may not be split across all paths.
+	Paths []string
```

Comment:
This requires >= 1 path supplied.
Also you may want to explain what's the use case people providing multiple paths (that hopefully will make it obvious people shouldn't provide e.g. 10,000 paths).

Reply:
```
	// Most of the time, providing exactly one path is sufficient. If the data should be spread across multiple
	// drives, then providing multiple permits that. The number of provided paths should be a small number, perhaps
	// a few dozen paths at most. Providing an excessive number of paths may lead to degraded performance.
```